src/js/node/url.ts

```diff
@@ -111,7 +111,8 @@ function urlParse(
   parseQueryString?: boolean,
   slashesDenoteHost?: boolean,
 ) {
-  if (!urlParseWarned && !lazyUtil().isInsideNodeModules()) {
+  const showDep = process.noDeprecation == null ? true : !process.noDeprecation;
```

Comment:
this reads `process.noDeprecation` twice when we don't need to.
```suggestion
  const noDeprecation = process.noDeprecation;
  const showDep = noDeprecation == null ? true : !noDeprecation;
```